### DISCUSSION TITLE
I can't use the alpha version on win10

### DISCUSSION THREAD

**Discussion by Hoinor** [Chia-Network/bladebit#212] [General]
`.\bladebit.exe -t 40 -f xxxxxxxxxxxxxxxxx -c xxxxxxxxxxxxxxxxxxxxxxx diskplot --cache 32G -t1 D:\xch F:\`
I have 64G of memory and CPU x 2
This is the command I run, sometimes it stops when the following prompt appears
```
[Bladebit Disk Plotter]
 Heap size      : 3.37 GiB ( 3447.82 MiB )
 Cache size     : 32.00 GiB ( 32768.00 MiB )
 Bucket count   : 256
 Alternating I/O: false
 F1  threads    : 40 
 FP  threads    : 40 
 C   threads    : 40 
 P2  threads    : 40 
 P3  threads    : 40 
 I/O threads    : 1
 Temp1 block sz : 4096
 Temp2 block sz : 4096
 I/O metrices enabled.
 Allocating memory
```

There are other times when he seems to be working normally, but then it stops working when it comes to table2

```
[Bladebit Disk Plotter]
 Heap size      : 3.37 GiB ( 3447.82 MiB )
 Cache size     : 32.00 GiB ( 32768.00 MiB )
 Bucket count   : 256
 Alternating I/O: false
 F1  threads    : 40
 FP  threads    : 40
 C   threads    : 40
 P2  threads    : 40
 P3  threads    : 40
 I/O threads    : 1
 Temp1 block sz : 4096
 Temp2 block sz : 4096
 I/O metrices enabled.
 Allocating memory
WARNING: Forcing warm start for testing.
Warm start: Pre-faulting memory pages...
Memory initialized.

[Global Plotting Config]
 Will create 1 plots.
 Thread count          : 40
 Warm start enabled    : false
 NUMA disabled         : false
 CPU affinity disabled : false
 Farmer public key     : xxxxxxxxxxx
 Pool contract address : xxxxxxxxxxx

Generating plot 1 / 1: xxxxxxxxxxx

Started plot.
Running Phase 1
Table 1: F1 generation
Generating f1...
Finished f1 generation in 15.75 seconds.
Table 1 I/O wait time: 0.00 seconds.
 Table 1 Disk Write Metrics:
  Average write throughput 2725.95 MiB ( 2858.37 MB ) or 2.66 GiB ( 2.86 GB ).
  Total size written: 33278.57 MiB ( 34895.11 MB ) or 32.50 GiB ( 34.90 GB ).
  Total write commands: 513.

Table 2
```

Can someone tell me what is causing this？

**Comment by harold-b** [comment]
Are you using powershell as the terminal to run this? I've had a report of it appearing to wait for input on powershell. If you are running under powershell, can you try it on the classic windows command prompt?

**Reply by Hoinor** [reply]
I run under CMD and still can't work normally.
`[Bladebit Disk Plotter]
 Heap size      : 3.37 GiB ( 3447.82 MiB )
 Cache size     : 32.00 GiB ( 32768.00 MiB )
 Bucket count   : 256
 Alternating I/O: true
 F1  threads    : 26
 FP  threads    : 26
 C   threads    : 26
 P2  threads    : 26
 P3  threads    : 26
 I/O threads    : 1
 Temp1 block sz : 4096
 Temp2 block sz : 4096
 I/O metrices enabled.
 Allocating memory
`

**Reply by Hoinor** [reply]
Do I need to close chia client？

**Reply by harold-b** [reply]
How much RAM does your system have?

**Reply by Hoinor** [reply]
64G

**Reply by harold-b** [reply]
Seems like it should be enough given the amount of cache you're giving it. I would try without any cache at all first to see if there's anything going on with memory

**Reply by Hoinor** [reply]
![微信截图_20220915122638](https://user-images.githubusercontent.com/30286118/190314423-3ef6324e-b15e-4814-97e1-74a0d15bf4a9.png)
Tried not to use memory, still not working。

**Reply by harold-b** [reply]
Does it end up crashing or just stalls there forever?
Does anything happen if you press any keys?

Also, which build is this?

**Reply by Hoinor** [reply]
![微信截图_20220915123345](https://user-images.githubusercontent.com/30286118/190315215-89ba6bb9-2f27-45b8-9b8f-8f94f98706c5.png)


Like this.
[v2.0.0-alpha2](https://github.com/Chia-Network/bladebit/releases/tag/v2.0.0-alpha2)

**Reply by Hoinor** [reply]
without pressing any keys

**Reply by harold-b** [reply]
It does look like it crashes, then. Can you check the event viewer to see if there's anything reported there?

**Reply by Hoinor** [reply]
![微信截图_20220915125214](https://user-images.githubusercontent.com/30286118/190317416-72690ebd-89f4-4c62-a446-146fc4bf7c41.png)
I don't know if this is the case, but I got a warning after running it and I don't know what it means

**Reply by harold-b** [reply]
Looks like you're also getting the `bad_module_info` error, which is a system error (not an application bug). [Another user had the same issue here](https://github.com/Chia-Network/bladebit/issues/201).

I am not sure what he did to get his system working, but you might want to look at that thread for some info and try running also the `memtest` command I specified there.

Here's an article which mentions common causes, although specific to games, which might help you diagnose the issue: https://www.technewstoday.com/bad-module-info-error/#:~:text=Any%20Steam%20Games-,What%20is%20bad_module_info%3F,games%20to%20crash%20mid%2Dsession.

**Reply by Hoinor** [reply]
I will try it as soon as possible, thank you for your reply, I will leave a message below when I have news

**Reply by Hoinor** [reply]
```
Microsoft Windows [版本 10.0.19042.1889]
(c) Microsoft Corporation。保留所有权利。

C:\Users\Hoin\Desktop\ba>bladebit -t 20 memtest -s 32G
Size   : 32768.00 MiB
Threads: 20
Passes : 1
Allocating buffer...

Fatal Error:
VirtualAlloc failed.

C:\Users\Hoin\Desktop\ba>bladebit -t 20 memtest -s 30G
Size   : 30720.00 MiB
Threads: 20
Passes : 1
Allocating buffer...
^C
C:\Users\Hoin\Desktop\ba>bladebit -t 20 memtest -s 10G
Size   : 10240.00 MiB
Threads: 20
Passes : 1
Allocating buffer...

C:\Users\Hoin\Desktop\ba>bladebit -t 20 memtest -s 10G
Size   : 10240.00 MiB
Threads: 20
Passes : 1
Allocating buffer...
Starting Test

Copied 10240.00 MiB in 0.57 seconds @ 17808.70 MiB/s (17.39 GiB/s) or 18674 MB/s (18.67 GB/s).


C:\Users\Hoin\Desktop\ba>bladebit -t 20 memtest -s 32G
Size   : 32768.0
C:\Users\Hoin\Desktop\ba>bladebit -t 20 memtest -s 32G
Size   : 32768.00 MiB
Threads: 20
Passes : 1
Allocating buffer...
Starting Test


C:\Users\Hoin\Desktop\ba>bladebit -t 20 memtest -s 28G
Siz
C:\Users\Hoin\Desktop\ba>
C:\Users\Hoin\Desktop\ba>
C:\Users\Hoin\Desktop\ba>
C:\Users\Hoin\Desktop\ba>
C:\Users\Hoin\Desktop\ba>bladebit -t 20 memtest -s 28G
Siz
C:\Users\Hoin\Desktop\ba>bladebit -t 20 memtest -s 28G
Size   : 28672.00 MiB
Threads: 20
Passes : 1
Allocating buffer...
Starting Test

Copied 28672.00 MiB in 1.68 seconds @ 17076.83 MiB/s (16.68 GiB/s) or 17906 MB/s (17.91 GB/s).


C:\Users\Hoin\Desktop\ba>bladebit -t 20 memtest -s 32G
Siz
C:\Users\Hoin\Desktop\ba>bladebit -t 40 memtest -s 32G
Size   : 32768.00 MiB
Threads: 40
Passes : 1
Allocating buffer...

C:\Users\Hoin\Desktop\ba>bladebit -t 40 memtest -s 28G
Size   : 28672.00 MiB
Threads: 40
Passes : 1
Allocating buffer...
Starting Test

Copied 28672.00 MiB in 16.45 seconds @ 1742.87 MiB/s (1.70 GiB/s) or 1828 MB/s (1.83 GB/s).


C:\Users\Hoin\Desktop\ba>
C:\Users\Hoin\Desktop\ba>
C:\Users\Hoin\Desktop\ba>.\bladebit.exe -t 40 -f xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx -c xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx diskplot -a --cache 28G -t1 D:\xch\ \\192.168.1.19\g\
[Bladebit Disk Plotter]
 Heap size      : 3.37 GiB ( 3447.82 MiB )
 Cache size     : 28.00 GiB ( 28672.00 MiB )
 Bucket count   : 256
 Alternating I/O: true
 F1  threads    : 40
 FP  threads    : 40
 C   threads    : 40
 P2  threads    : 40
 P3  threads    : 40
 I/O threads    : 1
 Temp1 block sz : 4096
 Temp2 block sz : 4096
 I/O metrices enabled.
 Allocating memory

C:\Users\Hoin\Desktop\ba>
```

memtest can run when allocating 28G, but still can't run it, I will try to update the system tomorrow
